textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriter.java

```diff
@@ -61,10 +63,18 @@ public void open() throws Exception {
      */
     @Override
     public ITuple getNextTuple() throws Exception {
-        List<String> queryStrings = Arrays.asList(searchQuery);
-        IField[] iFieldResult = {new ListField(queryStrings)};
-        itupleResult = new DataTuple(SCHEMA_QUERY_LIST, iFieldResult);
-        return itupleResult;
+
+        boolean NOT_OPEN = (fuzzyTokenizer == null);    //Ensures QueryRewriter is open before calling getNextTuple
```

Comment:
@chenlica Fixed in https://github.com/TextDB/textdb/pull/83/commits/5433e5ffba44d233eddd40956882193a23e0fcc5